samples/frameworks/react-native-expo/ts/messaging/README.md

```diff
@@ -135,92 +126,106 @@ Add a `src` directory at the root of the project and add the following files
 ### Add dependencies
 
 We need to add JavaScript Client SDK libraries for Event Hubs and Service Bus as dependencies.
-We use `@babel/plugin-proposal-async-generator-functions` to help transform async iterator usage.
-We also use `babel-plugin-inline-dotenv` to help load secrets from our `.env` file while developing.
 
 ```shell
-yarn add @azure/event-hubs @azure/service-bus
-yarn add --dev babel-plugin-inline-dotenv @babel/plugin-proposal-async-generator-functions
-```
-
-Then add the following into `babel.config.js` to enable the plugins
-
-```diff
-    presets: ["babel-preset-expo"],
-+    plugins: [
-+      "@babel/plugin-proposal-async-generator-functions",
-+      ["inline-dotenv", { unsafe: true }],
-+    ],
+npm add @azure/event-hubs @azure/service-bus
 ```
 
 ### Add connection strings to .env file
 
 Create a `.env` file in the project directory. Retrieve your connection strings from Azure portal and add them to the `.env` file. Since this file contains secrets you need to add it to the ignore list if your code is committed to a repository. We also add variables for messaging entity names.
 
-**Note** We use connection string directly here for testing purpose. You should consider the trade-off between security and convenience and better use a backend to dynamically provide secrets to only authenticated users.
+**Note** We use connection string directly here for testing purpose. You should consider secure ways of storing sensitive information as recommended by React-Native/Expo. Please see https://docs.expo.dev/guides/environment-variables/#reading-environment-variables-from-env-files for more information.
 
 ```
 # Service Bus
-SERVICEBUS_CONNECTION_STRING=
-QUEUE_NAME=
+EXPO_PUBLIC_SERVICEBUS_CONNECTION_STRING=
+EXPO_PUBLIC_QUEUE_NAME=
 
 # Event Hub
-EVENTHUB_CONNECTION_STRING=
-EVENTHUB_NAME=
-CONSUMER_GROUP_NAME=
+EXPO_PUBLIC_EVENTHUB_CONNECTION_STRING=
```

Comment:
Could we switch to Entra ID auth instead?

Reply:
The identity package doesn't work in React-Native.  It would be too involved to use OAuth2.